CppCoreGuidelines.md

```diff
@@ -12144,23 +12144,23 @@ References are never owners.
 
 The names are mostly ISO standard-library style (lower case and underscore):
 
-* `T*`			// The `T*` is not an owner, may be `nullptr` (Assumed to be pointing to a single element)
-* `char*`		// A C-style string (a zero-terminated array of characters); can be `nullptr`
-* `const char*`	// A C-style string; can be `nullptr`
-* `T&`			// The `T&` is not an owner, may not be `&(T&)*nullptr` (language rule)
+* `T*`			// The `T*` is not an owner, may be null; assumed to be pointing to a single element.
+* `char*`		// A C-style string (a zero-terminated array of characters); may be null.
+* `const char*` 	// A C-style string; may be null.
+* `T&`			// The `T&` is not an owner, must not be `*static_cast<T*>(nullptr)` (language rule; there are no "null references").
```

Comment:
I see. Yes, perhaps. Consider this an incremental improvement :-)

Maybe we can change the last line around and say something like "`T&` is not an owner and can never be a "null reference" (i.e. `*static_cast<T*>(nullptr)`); references are always bound to objects".